docs/tables/base.py

```diff
@@ -12,29 +13,66 @@
 
 
 class BaseTabler(abc.ABC):
+    """Base class for tablers.
+
+    Attributes:
+        alignment (list[str] | None): Optional alignment of table columns.
+    """
+    # Class attributes
+    alignment: list[str] | None = None
+
     def __init__(
         self,
-        template_id: str
+        template_id: str,
+        format: str = "csv",
```

Comment:
Should this default to "markdown" as per the docstring? or should the docstring change?

Also you could use the type `Literal["markdown", "csv"]` instead of `str`

Reply:
good suggestion. I have added an alias and way to extract the args to perform runtime checks still.